### DISCUSSION TITLE
How do I create a copy keybind to copy text in mouse mode when copy_on_select is false?

### DISCUSSION THREAD

**Discussion by MarkKoz** [zellij-org/zellij#3904] [Q&A]
There's an outdated answer about creating a keybind with the `Copy` action: https://github.com/zellij-org/zellij/issues/1494#issuecomment-1156595687

However, that action is no longer documented. I presume it's been removed, though I did not see mention of that in the changelog. Regardless, I tried adding the following keybinds:

```kdl
keybinds clear-defaults=true {
    normal {
        bind "Ctrl c" {Copy; }
    }
    
    scroll {
        bind "Ctrl c" {Copy; }
        // ...
    }

    // ...
}
```

This did not work. I select the text and press the keybinds, but the selected text is not put into my clipboard.

For what it's worth, the following do work:
- Highlighting the text and using `ctrl + c` when `mouse_mode` is `false`
- Highlighting the text with `shift` held and using `ctrl + c` when `mouse_mode` is `true`
- Highlighting the text when `mouse_mode` is `true` and `copy_on_select` is true

But none of those options are to my liking.

**Comment by imsnif** [comment]
This works for me:

```kdl
    shared {
          bind "Ctrl a" { Copy; }
    }
```

With `copy_on_select false`. Though you need to make sure `mouse_mode` is true, otherwise mouse signals are not sent to Zellij at all and there's not much we can do about copying.
